app/views/two_factor_authentication/options/index.html.erb

```diff
@@ -14,40 +14,34 @@
       method: :post,
       url: login_two_factor_options_path,
     ) do |f| %>
-  <div class="margin-bottom-4">
-    <fieldset class="margin-0 padding-0 border-0">
-      <% if @presenter.label.present? %>
-        <legend class="margin-bottom-2">
-          <%= @presenter.label %>
-        </legend>
-      <% end %>
```

Comment:
Previously, I was under the impression this logic was shared with the MFA setup view, but that logic is separate and uses a hard-coded label:

https://github.com/18F/identity-idp/blob/cd7c928f0d5d1613fd106376522b6bde25477c32/app/views/users/two_factor_authentication_setup/index.html.erb#L24-L25

Arguably, I think the use of a fieldset + label is a good idea to create a labelling association for the radio buttons. One option we could consider is to keep the markup, but "show" a visually-hidden, screen-reader-only text label.

Reply:
Hm, so this was already pretty broken as it is, since the resulting markup was a `<fieldset>` without a `<legend>`. Technically this is valid, but it's not very useful or accessible without a legend (related: [H71](https://www.w3.org/WAI/WCAG21/Techniques/html/H71)). I do think the keeping the `<fieldset>` at a minimum might be better than what I've done here, but I might see if I can introduce the visually-hidden `<legend>` idea to create that association.

Reply:
Thanks this is helpful!